apps/docs-app/docusaurus.config.js

```
@@ -32,6 +32,13 @@ const DOCUSAURUS_BASE_URL = process.env.DOCUSAURUS_BASE_URL ?? '/docs';
 /** @type {import('@docusaurus/types').Config} */
 const config = {
   baseUrl: '/',
+  future: {
+    // Webpackbar currently trips webpack's ProgressPlugin schema validation
+    // in this workspace. Use Docusaurus' Rspack path instead.
+    experimental_faster: {
+      rspackBundler: true,
+    },
+  },
```

Biome (2.4.10)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)